codereview-guidelines.qmd

```diff
@@ -0,0 +1,70 @@
+# Guidelines {#sec-review-guidelines}
+
+Adapted from:
+
+- ThoughtBot's Playbook section on [code review](https://github.com/thoughtbot/guides/tree/main/code-review) under a [CC Attribution License](https://creativecommons.org/licenses/by/3.0/)
+- Monica Gerber (MG)
```

Comment:
```suggestion
- R-Ladies Seattle presentations by [Carrie Bennett](https://drive.google.com/file/d/1rvS7nvlQnVwHfvLFVogTVY2OQ0QFCRcV/view) and [Lauren Wolfe](https://drive.google.com/file/d/1vdVCNpPWutKmoXfHbhqV_5s3UmCAPGil/view)
- SE Radio 400: [Michaela Greiler on Code Reviews](https://se-radio.net/2020/02/episode-400-michaela-greiler-on-code-reviews/) (Interviewed by Felienne)
```
Thanks for the credit but it feels like poor form to put my name here when the document is not publicly available and also I adapted this from other resources as well. I'd love to credit those folks instead.

Reply:
I know this is a public repo, but i'd be surprised if anyone but us is looking at it - the MG was purely to note that it was not from the toughtbot guide